Makefile

```diff
@@ -61,10 +61,17 @@ kubeconfig-path: $(CONFIG_JSON_FILE)
 	fi
 	@echo $(KUBECONFIG_PATH)
 
-validate-cluster-up: $(CONFIG_JSON_FILE)
+validate-kubeconfig-size: $(CONFIG_JSON_FILE)
```

Comment:
@neolit123 
This change doesn't work on my local k/a

I managed to get this working by adding
```
      echo "Kubeconfig file:"
      cat ${CLUSTER_DIR}/kubeconfig.json

      if [ ! -s "${CLUSTER_DIR}/kubeconfig.json" ]; then \
        echo "The kubeconfig file is emtpy!" > /dev/stderr; \
      fi
```
[here](https://github.com/kubernetes/kubernetes-anywhere/blob/85972f115840fab2f3a91d5b3105ee05857389b0/phase1/gce/do#L65)

Nb. not sure if echoing the kubeconfig file raises security concerns...

Reply:
it does raise a security concern at least until the cluster is auto-destoyed so better not to print it.
i will try again fixing this later.